scripts/sct_propseg.py

```diff
@@ -297,11 +297,11 @@ def get_parser():
     parser = get_parser()
     arguments = parser.parse(sys.argv[1:])
 
-    input_filename = arguments["-i"]
+    fname_data = arguments["-i"]
```

Comment:
actually... one reason for NOT doing it is if user have space in their path: this will break the code. So what should we do?